src/package.json

```diff
@@ -85,16 +85,28 @@
       }
     ]
   },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/q-masters/vsqlik.git"
-  },
-  "author": "phl-coding",
-  "license": "MIT",
-  "bugs": {
-    "url": "https://github.com/q-masters/vsqlik/issues"
-  },
-  "homepage": "https://github.com/q-masters/vsqlik",
+  "scripts": {
+    "copyhtml": "copyfiles ./projects/extension/**/*.html ./out",
```

Comment:
as i see this, i think we dont need this anymore, this copy all html files from ./projectes/extension, as i remember right this was implemented on the beginning where the settings webview was born and was a simple html file with native inline javascript.

Since we switched to angular this should be obsolet.